4_cv_basics/3.5_Interpolation_&_Resizing/Bi_Linear/Bi_Linear.cpp

```diff
@@ -0,0 +1,109 @@
+#include <opencv2/opencv.hpp>
+#include <stdio.h>
+#include <cmath>
+
+using namespace std;
+using namespace cv;
+
+int main(int argc, char** argv)
+{
+	if (argc != 2) 
+	{
+		printf("usage: Nearest-Neighbour.out <Image_Path>\n");
+		return -1;
+	}
+	
+	Mat image;
+
+    image = imread(argv[1], IMREAD_COLOR);
```

Comment:
Use proper formatting.

Reply:
In many places formatting in not proper, maintain proper formatting throughput the code.